tests/cli/workflow/nf_analysis/test_cli_start.py

```diff
@@ -7,14 +7,15 @@
 
 from cg.cli.workflow.base import workflow as workflow_cli
 from cg.constants import EXIT_SUCCESS, Workflow
+from cg.meta.workflow.analysis import AnalysisAPI
 from cg.meta.workflow.nf_analysis import NfAnalysisAPI
 from cg.models.cg_config import CGConfig
 from tests.cli.workflow.conftest import mock_analysis_flow_cell
 
 
 @pytest.mark.parametrize(
     "workflow",
-    [Workflow.RAREDISEASE, Workflow.RNAFUSION, Workflow.TAXPROFILER, Workflow.TOMTE],
+    Workflow.get_nf_workflows(),
```

Comment:
I think this is why the tests are failing. Maybe rollback this line to the previous version. See https://github.com/pytest-dev/pytest-xdist/issues/432

Reply:
or
```suggestion
    sorted(Workflow.get_nf_workflows()),
```
but I would prefer leaving it as it was before, it is more explicit

Reply:
ok I reverted it for now until we have decided on an approach about how to do this